.travis.yml

```diff
@@ -2,9 +2,8 @@ sudo: false
 language: go
 
 go:
- - 1.7.x
- - 1.8.x
- - 1.9.x
```

Comment:
This was an artifact from vendoring the main prometheus repo for the health target status constants. Since we're going to maintain that data in the API package instead, I've reverted this change.

Reply:
Can you please revert any change to this file? It seems that the indentation has changed which isn't wrong but unnecessary.